basics/github_desktop.md

```diff
@@ -0,0 +1,57 @@
+# GitHub Desktop
+
+A simple, yet powerful UI based tool for both git and GitHub that may be useful for all or part of your workflow.
+
+If you already have a git tool you like, or prefer command line tools, this may not be ideal for you. However,
+it is a great starting point if you are looking for a tool to allow you to easily get started with our GitHub Flow
+workflow.
+
+The [online documentation](https://docs.github.com/en/desktop) is more likely to remain up to date than anything we create here, so we'll just cover the
+basics of the sections likely to be most of use here.
+
+## Cloning
+
+The process of making a copy of a repository from GitHub (or any git source, but we'll focus on GitHub for the
+remainder of this document) to your local computer is called cloning.
+
+Cloning a repository is the first step in making local changes to files in a repository.
+
+[How to clone in GitHub Desktop](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-a-repository-from-github-to-github-desktop).
+
+## Syncing your local repo
+
+Before you make a branch, you'll always want to synchronize changes from GitHub as your local copies will not automatically stay synchronized.
+
+https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/keeping-your-local-repository-in-sync-with-github
+
+## Creating a branch
+
+[Branching](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/making-changes-in-a-branch) is a core concept in git. Every time you want to make a change, you'll want to create
+a branch.
+
+You'll make any changes you'd like to introduce to the repository in a branch. This may include
+adding, deleting, and editing files.
+
+## Making changes
+
+Once you are in a branch, you use whatever editor you want to make the changes.
```

Comment:
Something about the phrase "in a branch" makes me pause - maybe "Once you have checked out a branch", or "Once you have selected a branch in GitHub Desktop" ?